tutorials/preprocessing/05_background_projectors.py

```diff
@@ -0,0 +1,228 @@
+# -*- coding: utf-8 -*-
+"""
+.. _projectors-background-tutorial:
+
+Background: projectors and projections
+======================================
+
+.. include:: ../../tutorial_links.inc
+
+This tutorial provides background information on :term:`projectors <projector>`
```

Comment:
Note that most EEG folks probably have *never* heard of the term "projectors"!

Reply:
@jona-sassenhagen what do you think should change (if anything) to address that?  should we add a sentence telling EEG-only users to skip ahead to some other tutorial?  Projectors can be used with EEG-only data...  maybe a sentence here giving the high-level takeaway ("projectors are used for..." would be enough)?